irohad/torii/impl/query_service.cpp

```diff
@@ -40,29 +40,29 @@ namespace torii {
     });
   }
 
-  void QueryService::FindAsync(iroha::protocol::Query const& request,
-                               iroha::protocol::QueryResponse& response) {
+  void QueryService::FindAsync(iroha::protocol::Query const &request,
+                               iroha::protocol::QueryResponse &response) {
     // Get iroha model query
     auto query = pb_query_factory_->deserialize(request);
+    auto hash = iroha::hash(**query).to_string();
```

Comment:
Should not we check here if deserialization returned value of none?

Reply:
Definitely, at least `BOOST_STATIC_ASSERT`

Reply:
Well, I agree but do you think it should be done in this pr's scope?